src/Frontend/Core/Header/Header.php

```diff
@@ -352,6 +352,20 @@ public function parse(): void
             $this->meta->addMetaData(MetaData::forName('robots', 'noindex, nofollow'), true);
         }
 
+        /**
+         * @remark only for SumoCoders
+         *
+         * Because ENV variables are not passed to CGI due to suExec.
```

Comment:
Yes, because this code will add it to the MetaCollection, it is used to parse the meta tags in the frontend